.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- Provide a general summary of your changes in the Title above -->
```

Comment:
State somewhere what should be the maximum length of PR title and commit message title.

Reply:
heh, and what would be the limit...? 😄 I've used word "short" now - is it ok...?

Reply:
I thought that these are truncated with `...` when too long, but I've checked and it seems that it's not the case anymore. But being concise in the title is always good :)